cypress/integration/tests/developer/applicationinventory/assessment/Perform_review_without_assessment.test.ts

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    hasToBeSkipped,
+    login,
+    preservecookies,
+    getRandomApplicationData,
+} from "../../../../../utils/utils";
+import { Assessment } from "../../../../models/developer/applicationinventory/assessment";
+import { ReportConfig } from "../../../../models/developer/reports/reportConfig";
+
+describe("Allow review without running assessment tests", { tags: "@tier1" }, () => {
+    before("Login and Create Test Data", function () {
+        // Prevent hook from running, if the tag is excluded from run
+        if (hasToBeSkipped("@tier1")) return;
+
+        // Perform login
+        login();
+    });
+
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+        preservecookies();
+
+        // Interceptors
+        cy.intercept("GET", "/hub/application*").as("getApplication");
+    });
+
+    it("Application review enabled and disabled for Admin user test", function () {
+        // Enable allow review without running assessment
+        let reportConfig = ReportConfig.getInstance();
+        reportConfig.enableReviewAssessment();
+
+        // Navigate to application inventory tab and create new application
+        const application = new Assessment(getRandomApplicationData());
+        application.create();
+        cy.wait("@getApplication");
+        cy.wait(2000);
+
+        // Perform application review
+        application.perform_review("medium");
+        cy.wait(2000);
+        application.verifyStatus("review", "Completed");
+
+        // Disable allow review without running assessment
+        let reportConfig1 = ReportConfig.getInstance();
```

Comment:
In the same test you don't have to create this instance twice .You can use the one created in line 46.